src/pages/home/report/ReportActionsView.js

```diff
@@ -165,17 +178,9 @@ class ReportActionsView extends React.Component {
             return;
         }
 
-        this.unreadActionCount = this.props.report.unreadActionCount;
-
-        if (this.unreadActionCount > 0) {
-            this.unreadIndicatorOpacity = new Animated.Value(1);
-            this.timers.push(setTimeout(() => {
-                Animated.timing(this.unreadIndicatorOpacity, {
-                    toValue: 0,
-                    useNativeDriver: false,
-                }).start();
-            }, 3000));
-        }
+        this.setState({
+            unreadActionCount: this.props.report.unreadActionCount,
```

Comment:
Similar to other comments already left, but why can we not use the props directly?